### DISCUSSION TITLE
Autostart using systemd not working

### DISCUSSION THREAD

**Discussion by jmorris644** [tagyoureit/nodejs-poolController#836] [Q&A]
I can run poolController just fine at CLI. But when systemd tried to run it it gets this error. I am running node 18.16.0

```
Error: Cannot find module 'timers/promises'
```

I am using the exact text as in the sample.

**Comment by rstrouse** [comment]
Is node 18.6 the default node version?  If you are using npm.  Type npm list.  If it is not the default type npm alias default 18

**Comment by jmorris644** [comment]
Here  is the result from the npm list
```
├── @influxdata/influxdb-client@1.33.2
├── @types/express@4.17.17
├── @types/extend@3.0.1
├── @types/multer@1.4.7
├── @types/node@16.18.14
├── @typescript-eslint/eslint-plugin@5.54.1
├── @typescript-eslint/parser@5.54.1
├── eslint-config-defaults@9.0.0
├── eslint-config-promise@2.0.2
├── eslint-plugin-import@2.27.5
├── eslint-plugin-node@11.1.0
├── eslint-plugin-promise@6.1.1
├── eslint-plugin-standard@5.0.0
├── eslint@8.35.0
├── express@4.18.2
├── extend@3.0.2
├── grunt-banner@0.6.0
├── grunt@1.6.1
├── jszip@3.10.1
├── mqtt@4.3.7
├── multer@1.4.5-lts.1
├── multicast-dns@7.2.5
├── node-screenlogic@2.0.0
├── node-ssdp@4.0.1
├── serialport@11.0.0
├── socket.io-client@4.6.1
├── socket.io@4.6.1
├── source-map-support@0.5.21
├── ts-node@10.9.1
├── typescript@4.9.5
└── winston@3.8.2
```

When I type in "npm alias default 18" I get an Unknown command 'alias' error. So maybe I misunderstood the action you wanted me to take?

**Comment by CodeSlinger69** [comment]
[pm2](https://github.com/tagyoureit/nodejs-poolController/wiki/Automatically-start-at-boot---PM2-&-Systemd) is much easier with nodejs, in my humble experience.

**Reply by jmorris644** [reply]
Maybe I will try that...
